rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/CreateQueueDirectives.scala

```diff
@@ -13,6 +13,7 @@ import CreateQueueDirectives._
 
 trait CreateQueueDirectives { this: ElasticMQDirectives with QueueURLModule with AttributesModule with SQSLimitsModule =>
 
+  // DeadLettersQueue currently not supported via http call due to current API limitations. Consider moving to JSON?
```

Comment:
Ah, true! So the JSON is the value of the `RedrivePolicy` attribute. Then we should probably parse that :)